test/integration/api_docs/services_controller_test.rb

```diff
@@ -88,7 +87,7 @@ def test_show_service_management
         # ignore stats endpoint
         name.to_s.match(name_or_path_regex) && name.to_s !~ /stats/
       }
-      assert_equal actual_backed_api_routes.length, JSON.parse(response.body)['apis'].each_with_object(Set.new, &select_endpoints).length
+      assert_equal actual_backed_api_routes.length, JSON.parse(response.body)['paths'].select{ |url| url.exclude?('stats') &&  url.include?('backend_')}.length
```

Comment:
I don't quite understand this, but fixing this test is simpler than this. Consider that with the new format, instead of having:

```
'apis': [
  {
    'path': '/path/to/resource.xml'
    ...
  }
]
```

what we have is:

```
'paths': [
  '/url/to/resource.xml': {
    ...
  }
]
```

Knowing that, you only need to adapt the test to this new structure:

```suggestion
      assert_equal actual_backed_api_routes.length, JSON.parse(response.body)['paths'].each_with_object(Set.new, &select_endpoints).length
```

and 

`path = api[0]`

At line `81`

Reply:
Agreed